web-ui/src/main/resources/catalog/components/edit/onlinesrc/partials/file-uploader.html

```diff
@@ -0,0 +1,58 @@
+<div data-ng-class="{'panel panel-default': panelMode}"
+     class="gn-file-uploader">
+  <div data-ng-if="panelMode" class="panel-heading" data-gn-slide-toggle>
+    <i class="fa fa-fw {{icon}}"></i>
+    <span data-translate="">{{title}}</span>
```

Comment:
The overview panel with the change displays the header `Download` instead of `Overview`

Reply:
Fixed.